.travis.yml

```diff
@@ -5,25 +5,28 @@ language: python
 python:
   - 3.5
   - 3.6
+  - 3.7
+  - 3.8-dev
+
+dist: xenial
+sudo: required
 
-.mixins:
-- &_xenial
-  dist: xenial
-  sudo: required
 jobs:
   fast_finish: true
   allow_failures:
   - python: 3.8-dev
   include:
-  - python: 3.7
-    <<: *_xenial
-  - python: 3.8-dev
-    <<: *_xenial
   - stage: Deploy
     name: Publishing a source dist to PyPI
     python: 3.6
     if: tag IS present
-    script: skip
+    before_install: skip
```

Comment:
```suggestion
    before_install: []
```

Reply:
I'm curious why?
`skip` works too, isn't it?

Reply:
Yes, it's just cleaner this way: the empty sequence of commands `[]` means run nothing while `skip` is a hack for `install`/`script` steps which require a command there. It's an internal no-op command in Travis CI.